physical/raft/raft.go

```diff
@@ -574,6 +574,7 @@ func (b *RaftBackend) CollectMetrics(sink *metricsutil.ClusterMetricSink) {
 	b.l.RUnlock()
 	b.collectMetricsWithStats(logstoreStats, sink, "logstore")
 	b.collectMetricsWithStats(fsmStats, sink, "fsm")
+	sink.SetGauge([]string{"raft_storage", "committed_index"}, float32(b.CommittedIndex()))
```

Comment:
Should we limit population of this metric to the leader?  It's the only one whose committed index matters I think, and having every node emit might make things more confusing for the consumer.

Reply:
The purpose of this change was to have a way, via telemetry, to gauge how far behind standby nodes are from the leader. Given that goal, I think it makes the most sense to emit it on all nodes.

Reply:
I guess I'm wary of using "committed_index" as the name for this, since usually when talking about a raft cluster that term has a specific meaning which is leader-centric: namely, the index of what's been committed with quorum on the cluster.

Reply:
I'm happy to change the name of this or even use a different metric entirely if there's a better way of accomplishing the goal. I'm open to suggestions.

Reply:
I'm reviving https://github.com/hashicorp/vault/pull/12166 and writing docs for it, would that work for you?

Reply:
Yes.

Reply:
Reverted here https://github.com/hashicorp/vault/pull/17397/commits/31f53501ad1949cd952fbff274db57845ab19444